### DISCUSSION TITLE
Search Environment in Activate & Manage panels

### DISCUSSION THREAD

**Discussion by stalingino** [Kong/insomnia#5749] [Ideas]
I'm working with around 100 environments, and it is very difficult to choose the environment i want to activate. Same goes for "manage environment" section.

Please add quick search options, such as:
once clicked on environment selection, we should be able to type which should filter the list of environments.

**Comment by erodrigues-dev** [comment]
Same problem here, feature removed on 2023.1.0 version, please this is very important usability feature.
